surfsense_web/hooks/useConnectorEditPage.ts

```
@@ -0,0 +1,240 @@
+        setIsSaving(true);
+        const updatePayload: Partial<SearchSourceConnector> = {};
+        let configChanged = false;
+        let newConfig: Record<string, any> | null = null;
+
+        if (formData.name !== connector.name) {
+            updatePayload.name = formData.name;
+        }
+
+        switch (connector.connector_type) {
+            case 'GITHUB_CONNECTOR':
+                const currentPatInForm = patForm.getValues('github_pat');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)